data_architect_misc/data_transformer/configs/NA_United_States/step4_combine_tranformed_data.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "__comment__": "Use this config AFTER running step 3 config file",
+    "__comment__": "IMPORTANT:: Make sure to update the name of the files in 'create_new_dataframe_from_input_files' function.",
+
+    "__comment__": "If we pass None for 'sheet_name' parameter in reading Excel files, pandas will read ALL the sheets.",
+    "__comment__": "The equivalent of None in JSON file is null. That's why we are providing null below to read all the sheets in the file.",
+    "header": 0,
+    "input_delimiter": "|",
+    
+    "rows_per_read": 2000000,
```

Comment:
Are you 100% sure that the total number of rows in these three files above is no more than 200K rows? Please make sure that is the case.